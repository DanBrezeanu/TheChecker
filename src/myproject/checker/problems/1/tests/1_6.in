781346867
921949079
-43507613
660318122
-758239077
-738963780
696876987
979306362
725016258
-3965025
114405337
799574229
-67788164
566942805
787081159
658520833
375473741
394409053
-759343065
-554789076
579357597
-967635210
205841764
749858634
550485668
181153762
700138403
905328584
-150069463
73818365
-33838168
-95320923
-1949685
27459671
58592612
-447325778
-538307683
-341738915
735662286
-592463728
486750732
332736418
-819340066
-426526266
205784457
503254313
840466412
318590375
-551449008
-601439305
-492379322
-630192461
676343305
-185525837
-654341145
76064760
711118445
-203858630
194973059
-501035148
986438601
-459729233
472871408
577318022
53298096
430396770
-556508454
717667057
445567958
860575204
235385403
-504223395
392396707
-123200146
-520797913
-490126050
402544778
361226577
-948009934
763959663
-684418009
381285387
803372455
-869925493
-37928722
-392207980
-8901943
263950956
436193211
259462137
-706923947
372950912
-21499903
972409980
283188735
-273103193
-74302004
-206927027
851070682
78360078
-99884574
-756099677
524667003
943651307
-982490824
-961142634
702808320
283723654
865865288
-597472981
-433955353
674310232
-345637817
-685878148
-684607679
139895393
-28240486
491243495
-825475516
23848357
217607218
182557072
843005475
890384389
-365607747
-721673628
-972361848
924690795
-61191480
-113925992
-359987572
-58969267
-574091609
772598847
777171852
-807875192
-654567576
-388203817
915142236
200635689
774187177
745328553
-711530046
-674530516
651310362
-19422084
231768741
548834481
-323236269
180694746
-420082187
-228205374
-20233125
298584343
646837577
380646458
67690614
329311295
571145673
1995933
-947500464
680389298
628118569
-953557304
141415313
-169675689
512710901
-469750872
-418145525
-680704633
264132457
-286451087
179142789
348805127
-847753499
-837071951
-441712545
-146341414
94841863
241163312
-33052306
-249411427
-134942564
21395772
-610010253
108601431
151005859
112677581
-549297072
-121849633
-90801036
878017325
-987173428
-332633692
566622855
-864127121
938597961
356254740
994302228
683117370
-812574645
988762997
-831399350
-218777392
-495093425
803263755
-556060567
848827758
585953558
935351226
544578968
-254540898
-151146383
346207175
947623937
-654906541
364094832
-452929810
833123907
913125301
-728087909
179605835
589582063
43955935
-682260296
-9139789
-520324580
317551187
-245262353
895720346
-200785259
-632743197
-857317830
382647228
979624173
-956912534
-783537717
-41739453
20929344
64520764
541048880
-724808450
258712571
-607378034
147079152
752946671
457804121
961425132
860395124
-814306295
-257564791
76761092
-50618624
-108317666
253850552
-441064053
980405095
430350105
12748292
-935927601
-309569788
-426956192
243944545
-853327013
223546365
89249221
573904842
-116569733
56075208
254238552
825069015
-650123746
-661054262
-989382675
440042555
-730460345
-429715876
973841423
420765897
-907585065
674899935
52356742
-179778153
-202046601
-897712445
477653374
-799120410
-915888730
734131533
350943629
-794755743
954037415
-543871870
376957632
91835120
944145959
-335761399
34231475
597118607
708537488
-538263666
-290307148
92703673
975872759
702233695
-898678500
41763313
-120420240
407536333
717552828
-200880833
899990650
-415990309
903649906
814363498
-838036758
700535115
91101516
685237133
-974534964
705344793
926020334
-768602883
698702653
-571432497
258484560
652120919
44021443
-799693504
679673557
-551198082
-816743654
263313965
193105645
555079889
602166851
459085965
-162698066
-578058399
493162457
-661775441
-983447565
622272001
938340762
621653383
695978670
628915146
371646374
-452866044
178142440
913124494
-398481602
-87620872
856955452
-937235465
825534347
416596965
161097563
428689619
448820797
-204430891
-493700826
916381695
-197948818
218320612
-183947431
-617550731
687421695
242916
291542187
-854414416
566986149
-731915712
-52254991
429174710
-72790517
-247700707
-999111518
535150446
245430887
-171026661
-273053338
653411409
957203433
-618337924
-812891386
431912231
-864653437
408279855
823392633
-38344693
-767715785
-892168081
-761034775
-38486718
-120978648
-893172164
-934582343
206858856
970980911
115570211
896520834
-254258512
232443584
-451430472
449346053
-711367895
-291308073
-652404709
320362829
-444083792
-849785872
441809726
-652411380
-571988907
364998026
-860414375
913128124
877728914
238363466
-3460907
-771381425
-349230905
197898506
-263201619
-19570990
777985027
-992042937
544125724
-235672623
755683153
449066846
10389115
609735871
-152022429
-515653447
303026430
442708017
817201213
559554675
-389190587
220106099
-382359201
57591188
476406357
784718169
22441486
380989409
566338510
-842228989
946754042
880029205
426045253
-161401183
-435055460
-814250077
244851563
6035686
-725069995
-453306228
-862624875
343855212
201853747
-900810123
-903674709
372484901
669573031
-925381349
106784770
-946377103
401966738
102498496
-782386830
-880321052
925773865
457308757
501392203
942395319
-663046199
-837148323
-903410219
453033771
191083331
84468558
-365869990
-893745437
501780384
377881018
-628091025
-567189402
128903072
-171597010
-152222221
76218928
264019085
-455253114
-479231627
657652689
-508867297
372217902
940457421
148160492
-727385109
447448547
59188588
-690673178
430206211
-198079731
459333538
-671394103
558244706
579330836
58441861
-709102830
5119947
-740311817
-45900281
819703407
-688484234
374272851
-905703449
783631104
-758788929
-285804599
-152703186
720838042
630369404
-321533070
-695353743
-162910964
625161790
221445995
349053296
154722647
78225896
-197736171
-185935814
537750265
786512586
248241435
-630164503
-4130549
9863708
-632597229
137920804
893968367
-254076416
409827286
515560494
-625765452
926940353
870634666
-795187197
-88149220
-435992664
32100872
503866667
-648448294
-401102867
33010215
-264973691
935754552
583647087
802668601
764994078
707743881
-234292572
71940522
-306321505
-803169077
-262823134
157914931
508166768
623459737
-58553166
292073091
-399414235
-575816764
-831659346
967086587
-277913683
-14345737
727040479
139956672
366552297
778239384
646333166
61831325
-640002212
-858640766
56248104
-256970813
289978143
-656643572
570500651
-655411533
-870203668
520770414
-406569239
371278534
198939149
-106643545
-239222784
-193314534
520458904
-732795772
617939370
301763206
705307056
-356045642
371178660
542536240
-254680501
948410426
-228052462
719959502
268473308
160903783
547204895
-258769842
-658461726
-119964185
-996440804
-188237117
876435826
351417482
84352665
-653864035
184795488
893860786
261545938
58244205
474895780
-734596217
-263919859
4171701
-791750695
555452727
-248657563
413166497
-471415549
618893553
36353258
133125588
-199675245
543691344
-749283520
-435986377
487662954
-732903238
-199087236
-124766854
-889210719
781588876
754568371
-158298644
-945977301
600493204
755466776
73611421
476432541
166559141
-234420039
990386056
-430858000
168138610
-554894571
-994167545
-616879650
-41971624
-428408545
-224614634
840995146
315011827
355917
-992661409
45239549
791963212
312658110
-879307230
-106362098
620345334
-543881451
-735640341
87091169
-223247710
-803954551
678010568
-13811078
-244169383
-290432866
898438424
488241142
545381984
-863539428
206391441
221444094
270948313
523329139
403960292
291895950
-882950280
-703923504
579203271
381449938
-705982660
-78774997
444340122
-443216543
-845919113
989057013
244576681
-364498042
828225124
136630578
-445264059
254009039
444115693
-603657052
-729312902
-985072149
390640712
578617855
-682665261
-210340080
306102988
368249792
193437248
181673466
307448721
698784218
-82065212
-238865791
918156236
871394937
875607152
780006054
221865374
331813822
395530800
-828407215
737786741
553629645
738000970
-434195504
-992066037
-999938132
-919981918
999017071
561837808
-290185042
-773789008
-434323290
-583327543
-654331520
-213412984
-247136791
681440330
-600092579
-267666222
-833979857
-514563884
-144108416
350288177
-345263576
871253250
-766145639
618512535
167412269
842430688
670603864
138094864
429418547
752379606
852847564
-929437691
-585722675
34865745
730155217
-611157995
-324776818
-890969187
-48308778
-843737703
633687853
-282869568
-669254104
-668460451
-417666674
839735547
423652463
-606838487
78431625
-945701827
-662126739
156293226
-760898978
449564596
933045367
-960314099
-16086102
777916734
774057855
-326199330
224416995
944572321
536903821
-455637420
-772282567
-230963396
411394005
-503867935
-712147535
470185614
-458329853
431020398
647796819
-408996670
590601762
-457040895
532497109
126161137
797470746
-708461246
-162053849
666417893
301779359
875071560
-528378700
378787470
670532683
531997140
405386584
-909222679
567600109
61271434
-887738498
431057185
637399375
-380445616
686770577
-33893093
950245013
118824062
-968129261
873109274
-134540143
-831666886
-581710632
-909959555
-326814767
455614061
-722714506
743426166
42773670
-731078187
-69277931
791167412
906200872
931242608
-486683656
-384091526
649272118
416492202
-947004434
680137256
-159982112
422135844
-827992298
345337437
-37050542
-319599832
660437171
-923558583
833209919
357012374
-98485432
758994557
-428394280
425649692
5987629
-307307711
114822861
-220041633
250349493
-715502092
-895961659
-237057123
-860586125
-358935941
56197540
-555389652
611637166
-115091724
-197787284
-658576201
947268426
-502521340
208881386
-648489073
-884345659
421883357
-162130527
183088169
61695753
-126637015
-253883281
446875523
563705960
502813582
678890207
-158195592
-136947369
877570422
410925854
72399114
641939711
-822766733
-556434758
378938888
-510151863
-931279002
921967677
343017937
-240219628
-82847735
-427655250
-209692630
-531063402
-762652397
-637268438
-579234733
766078084
374993064
658143094
167575697
393946330
-694150751
940340627
600662217
-853558960
404229499
-51344347
115372147
196919907
-992663318
811651512
-472642687
-609582023
-38714065
-517750597
-191408397
-594074753
-169912558
-151285413
-391741037
-554755791
-773391806
-252104008
-429963480
-312791947
293186480
231773169
362145944
745935962
780194809
-524082680
-538903406
33000314
655062516
548566338
-17647939
-940967494
976102686
574132844
-805232950
317630182
-212048288
-625504357
-308960407
-952951996
52842447
922467321
293658162
712835806
5130297
-62030990
96175527
-364314275
-155868040
-539347734
648350527
567659379
651882029
-728961543
171403682
-849593931
-494026361
-904558591
-475289163
-317377326
57961534
-832230172
880737817
-149712830
895131953
303147621
888064197
-70490272
743676428
416076430
58362004
908747377
-812515503
-469109999
308052262
626278623
-214620676
128796612
-432233399
248010449
-265684917
-495842456
306451727
73513161
48263180
782855138
531654726
-169573498
474106473
256252900
398563444
18089988
-539030414
-496014380
-13563013
23764752
810303964
270223125
872679677
-224054669
743041514
845526756
-649229621
-780283296
223859178
137610725
-320934748
-651150131
21274370
-177693013
-228112672
853141993
749727787
-730462481
-913758567
983918165
-86195249
934717694
148894932
-824834205
-672919636
65659867
-530249339
66470225
-896919363
-483173191
321122226
-146302002
-286071754
129273284
741901135
-625135441
-526351612
-520808165
-486835070
-535719647
570469082
-278370412
9117248
441521748
-198828055
-994108506
281523878
649002520
470866474
-996107625
22282922
691631318
254886030
684619152
-81194751
-738160568
32592937
921592580
824790993
-976411142
542183577
-348007276
127956580
-845993470
-98623650
106752237
-605037146
198229617
-307973504
715652873
729640801
-590295428
-132880029
-177160475
68241557
834164244
-855954221
-857052994
667855096
-149965682
500937033
907141982
987423518
470277470
76362672
-300837210
204408872
269541576
-528249293
954940516
56152828
382010831
400266161
-589349772
877696124
62272297
155184127
94792793
-529740643
-86516200
-387684109
-781428737
-727152529
-638853445
-727465859
-175452195
288366684
-606902164
-346488553
238050178
-75120193
820095381
-699338682
-686017161
-474466851
-502276473
-40498900
-146749556
-90703657
139099459
-833569566
-296048737
452308405
992399524
-558356440
-268149049
-803055681
-683400916
53104482
488706212
779215116
-148877973
-496728752
145300389
-540002211
932202260
-93256624
-899150580
921416022
-688990606
599085492
764191922
112275048
-318151961
-321292379
782027867
-240298403
-99027057
721130273
895490429
351005401
652438813
138385861
339397477
887063105
-716254250
888912408
-451091057
-669639152
-127724203
-990963875
673559615
-517441084
90478018
-856200737
-265334524
-394898063
313155292
-792524436
37355067
-629644775
435534897
845403425
251209562
457934473
-588678809
-284099877
-599776991
-861513322
-332808645
378847935
-885119063
-786663311
-993022468
-438924595
884993279
-438268881
979196570
162375597
529716515
-70632216
897859161
615237773
-126725266
235419483
335731252
627214060
104586372
-791113682
346384652
-868524535
793536121
253501471
-454633695
200904938
-978225427
493280898
-340926150
2948109
928380483
517061135
-931602453
-91325223
507394028
353612902
-791732007
690811202
-55737262
303994714
137594586
-822675904
-280002317
330191694
-351815485
581886827
-742241755
661788302
-974203264
454955399
-474202975
-691743679
-108497370
-140972629
-289543189
-230479264
-585244296
-520382323
-892469726
-830448447
591688983
-287739403
-859056407
-5637868
670625267
667733894
842176028
339831722
314844030
-309828020
-379609894
-506227086
185984634
718177773
304151253
487040240
94607714
-419847006
-249169745
387704742
-61396691
-34614261
-247292796
-647320940
-765185969
514189081
850870405
904879731
283036813
205305421
198078856
-13523644
951016640
-974014007
-626667186
-356334619
-755132699
449315558
-353115461
677990000
691878646
-640999818
-996054036
453371005
-30558632
116238223
-387217109
172921934
-877962486
482052246
-831243064
-281011075
-166542152
243017772
-25719051
-870975076
111594335
-644431589
514362554
-77894062
796463376
-59809339
731399286
-569435992
-22195493
679758769
281768872
-180366895
-649190853
-823407422
454494549
710051193
553892493
480411382
482509732
-142383829
802066450
-766742986
-777986246
729770819
53262205
226537967
311245160
400269006
204102072
-726945692
387425040
-687660518
404831177
171387867
-332179386
279054485
56551671
-696996514
-73189789
-89400737
-945470071
647289414
-257737633
882286379
149238637
-394387970
-346120894
-591752560
364067198
411207780
-7882714
179613942
-627341170
754492795
323415345
-599963616
370350412
-686087675
383148581
-298182687
-73533148
838782747
56012411
-488886332
-643954370
997713064
29100849
668558136
771556813
-118742707
-596418246
4712549
893511529
-774600474
-820638274
-302697853
-164704237
353001001
905221246
-497522649
964794908
-301750390
-458157424
919449068
-403460798
531562062
-296961681
-643608370
324901013
79872012
303734765
-957215767
-966569364
462577471
6904716
-824306929
-864782919
-292748083
-372336356
-968006704
-592549833
-703634316
197929005
853496759
24488866
-79358928
-1766225
412219283
-339598304
-652150896
-983905176
414702014
-76717677
400743930
916453261
-942430946
-301253674
178361687
-149924239
-400801534
145407035
-81531082
-645511442
290945083
538355450
-664467577
-610267261
-126219130
599583795
781625573
272452197
-540056669
489056819
11084707
49263538
-162374378
-410742208
-583899323
-686743686
827024726
-291124623
192760160
-397628668
-360306814
187408599
156585229
58662820
-34392407
-958528118
382627637
742877592
-776921827
581910882
-166482786
-898623663
-323731356
890261201
380465835
-453945422
-521439298
86652947
-184411367
698328175
306817557
713924356
-854319310
151229773
-601975471
-462049567
-957806852
-939755963
-591848715
175907164
-748487693
147386362
868058684
-107793929
590578601
-499916814
-20818392
-670371861
544747704
251907441
901990741
-836343989
-512757539
635226063
629205152
108951323
-657206341
112220797
-319141434
-307894731
454916855
-66160534
764322306
-267100916
849601256
-964354431
-301694764
-873076049
398080177
-376437000
694116429
-600796096
-370676028
-60325630
52659149
-316091446
171864722
234370515
842313832
-675638130
-530991818
-369115637
-467147189
-612800834
-254096015
-973151638
-107189524
746531942
670881571
135670032
-408818634
-793929577
239526102
-477445021
-721321652
972441710
213105195
398072520
552451625
-193738937
-590865182
711445319
-877209082
-862051506
-600781054
-232045577
-315221154
-374088518
444988435
44943132
136613647
695548473
-284546328
43857876
264826680
-392060710
-122836918
623747593
546208997
-374972316
428973116
170662450
26784506
444434651
834767621
237265611
-408170224
85668996
-242807896
-160368100
645313922
-576571584
-252881460
719687980
-967087614
-320111725
272550575
-648226407
-2909793
-688218618
343126272
-445079884
-617531516
-865865581
-273140309
-789418655
611598526
-14155231
-845433789
-607677931
291590236
209384077
-140057442
153330695
105478701
-988643662
-854532314
129863134
-437135362
173933939
-964582464
327884155
-839813974
-685268507
-17617514
-395392993
-552825177
555741039
609871860
-905582701
-516829512
-323094488
195401282
179524306
-923524254
-211302033
323065867
110964742
-160795776
-715067577
949378683
216477592
-457772397
841514174
624130355
-268905272
-879494969
55755907
-813630440
489005241
-635438261
559431705
-823553114
-36271320
-72905402
567856990
798055339
634906004
-773358344
630443116
-31355459
472638099
309718115
-887523122
452547387
462801692
-146099771
284976764
-898904441
562430889
969279114
-673436004
377684055
-251881069
614913751
-331471025
-600161181
-779889052
-83467982
614643114
943569507
925074723
66992309
43006291
-70309825
666884244
248413429
-428630725
-865360398
-487204181
835370398
-791334927
971281932
11108253
-626656416
-564842633
137584001
707572956
386270340
-99446741
-124969459
-107559461
-727326206
-867949624
-454741572
907098213
300385855
962197155
-560212595
554381347
424148847
233942119
450252162
-933181524
463672073
-174775107
467628806
776609674
860567377
-322060601
-482208268
917745448
-445092773
381053409
-393744892
-556896749
-738318155
793726043
-512303442
-514575857
-715155524
82030948
8889784
-295766210
-932567461
-799348466
156941129
-844307201
199113729
214925921
449746212
818673841
-479445031
933863900
542756726
-450359296
971903944
353018850
800974981
-549416034
-985626176
495300048
-506063195
-879234444
754435356
-187184526
-856179054
492112177
-253469144
-146120716
969952142
932020501
-327319645
-203801885
147956066
294977326
93340839
-994129429
-510352664
-779684720
268695601
160608615
-421335421
-462364506
548950022
-248888775
-292130071
-623837153
412142935
209236681
-301533313
73721214
790652989
833891560
550163230
-446826888
177279671
-738221680
-540186744
-10681400
766166321
-312032593
-780524166
38298663
-305110323
-709023957
-715575501
626161469
229004476
-157077807
-989854989
-882506136
556249064
602384770
-327651940
-221340615
-116206865
400419850
874128721
-185090900
-728744377
911029471
-27896526
-651753611
-324393529
12358255
290416574
-368350510
297913165
-9538689
918509588
-515662010
248420884
-886304802
-268672790
-530491513
532171028
-814769776
-815219881
-237348050
905245433
963381918
-635168188
-235186671
276295144
835686595
-414367990
151889339
685616197
312582548
-820549928
-276004809
-630491997
864471115
880262804
-566055550
-782086902
172370315
126575313
605343517
-569887125
336823287
938022359
296717221
-216781795
-556344325
161898604
682896211
284800857
-518981556
702334231
811624076
-255686786
333333406
-533372476
338013860
671700061
482797230
-712455589
691732509
628876085
-925582683
550748870
139901279
950783866
-219797997
594561893
264455309
-282124814
733796268
-118173971
971532653
-94399967
-943697845
197073787
-132043292
897402282
-397397885
303254133
514890230
-935435049
-375126334
177543149
-975990102
64509447
715119820
271887744
-156825020
336185725
-983550128
-699567908
-554170886
636182479
-473979024
797596294
-956552997
-900049947
59521221
115449391
-885525915
831747373
-915062407
-309375591
202309792
291908510
559502092
900228534
-276396242
593702513
245735732
738639711
977980051
742728661
446231188
230126027
60535754
-655823746
-395612654
52694159
729731479
327395408
700990167
84578766
458891740
962315232
-573973294
309172143
-221042974
-648759429
345419565
-801789216
-473838986
-600839529
-489018082
-276125254
435248423
995536930
-69502099
752028095
-943296047
998812067
-524946963
-39619953
408231141
179280407
637197245
928810128
-697777322
-969256860
-752414107
335141311
195145544
-741955053
735965041
549532011
-670250155
129221995
720139428
-347588410
411748866
-470242973
-53083875
951760144
796031077
-141838785
-157539992
-312430775
468296734
-636720127
372838376
-398246216
-512452068
165743240
-129216000
-61284499
168539395
703508150
526288081
591264206
185855950
843295164
-82916934
454883117
896299076
-571919872
-424564289
186528165
540996891
472406620
-626517930
-422928943
775385039
273145546
966745118
-566093478
-30445369
-94084414
278597362
894150438
917858579
-316439769
959018265
989605622
380147338
-384648610
59950308
-482483039
-918709062
338316996
711828249
-159135972
-821486919
213351582
269859719
-891127432
-654412060
-117815185
-213035134
-468714134
-138136445
445457900
110755777
-873643308
606757785
391868182
-102598418
573450023
956629301
156075583
396560568
-28315977
-634903287
-972704773
886704443
-463013702
-740860845
757434351
-584958568
-463550041
272980271
630892440
-976707141
-707397420
-53718504
-447191959
-324127233
541350029
10424016
-849447848
-542824491
-687778910
608609640
981682663
-947606641
-122791140
-283125523
-93789777
146471036
289103506
-837657217
-7367260
153402600
-521894711
-687856016
-839552546
346919361
-464996134
22585841
-136725767
-3025367
-571464403
337740282
-603422357
208232495
-466856898
-688692693
-193624414
-370352831
-826140360
622330864
732665798
-28612385
-969928256
-923820948
363459874
-284776719
-942730905
596071989
-648554473
625039301
-55299401
-869621168
813052570
-174964151
83139333
801698052
-932517670
337245435
-765794273
705593223
-6569766
918608666
624858941
-906820492
-880508683
215083539
32444353
111108559
739868816
800000418
301448528
389558708
-663121242
335119058
-19766701
-845058852
496875288
-334463816
714253954
-826342328
-248458584
497208442
-432464535
-77021801
372717081
934696558
542530683
28766349
762845829
904722066
176034184
356185702
-159662177
91297796
506286146
-803773085
258510938
-395733388
-420102162
696574283
605920409
-703097002
189843685
849021864
-764894329
-476355037
947314610
302724388
435750135
577693457
247159041
-537164163
806751699
334783639
-539822345
220600382
-335047894
719715041
-352859458
-114371509
103460785
648287667
525239767
461677413
840966447
897835543
529989787
-844796493
-675465447
-810150854
-46654229
463504436
555575453
-719669126
813029243
177167861
-265523955
-639563756
-168389751
916769077
568762847
-425223611
353134404
-495383780
843342794
-764334942
-515993152
523482488
-82452134
214633786
907763265
139202096
736500875
866010996
793675484
-358480893
677597910
-331953768
911023494
-978450285
-742302259
-54422342
803730425
334123176
-535721410
317465736
234583309
-709444847
-450718195
-940024920
753551441
-585808480
801303313
-314170386
-291134628
781243421
872582361
-42679724
-627448406
331485347
-53077463
-216244907
-795785038
760161346
-128137206
-171013487
285608452
844285847
102799226
90465024
269376462
-933680313
40356384
-782613157
894347609
25567874
996117285
-298303705
295291516
688447160
842119013
-862633729
759527415
-166507769
-965951787
999362959
405748334
795317830
918082835
761100816
376877619
802266152
428909889
169304799
601826906
-635255015
-972476256
-403682943
76837698
-650448465
673169874
805287553
-972095917
21110260
-534414511
266790674
144412938
234808088
490831596
-131411242
-242656165
159319303
-60373240
368257470
-875164374
-112301585
-945561646
-719370913
-915113373
-791032720
-758319630
-735666945
-965525796
67141326
578947863
278470896
-57652664
-222862362
-846586575
419394734
724734290
816688256
574488901
796992299
-245714115
700169063
-223827337
-715175662
812898052
-341056915
160570777
264133980
308234874
-470571829
-133970624
-794135155
151922815
-397101908
-948156221
-574363970
-960619264
-859375143
944341388
-970172913
558786346
326424093
-685573219
-722414119
708595433
5004677
677451223
-313904036
687968271
-390002995
-29850292
-673863860
-989887922
125034122
533488945
467148488
-777506200
423510452
793449973
-644361783
-174922494
-590275343
-24374964
-394096749
72455025
623251093
637230035
532862790
-737934999
233165089
336143180
-425243258
125946891
-158512031
-158621287
-79894168
869634571
9740411
839795645
644106458
795180780
-41166514
-459860816
514178004
-223499589
88526049
-480723255
-90350543
-832889010
760301848
-820355827
67985925
-189014938
827463567
537258791
-554229751
715639769
57878729
-507304477
-235174353
47707820
-41854098
675922129
811576691
289754685
-319108078
601693178
306555759
-624722898
870756445
503098094
56026000
244809508
-260144933
207055529
164153198
700313116
-590949627
-612350
431158464
-209507932
426060108
981234654
-617576331
-238044468
-548930814
672665267
304311886
-563841089
200817055
-666167651
-195498797
445944545
175814462
-529912509
424817518
-847948751
-398080336
504030759
-324655260
-75740956
10967601
-684684384
-156805137
-718800303
150299676
589592999
-974998729
777323074
-232206777
-336668464
820415964
696950631
-537231758
278548156
287113003
-283875150
911317526
317709365
-800520932
-209185083
-454443957
-146160993
159499454
749219476
728690582
-857100455
889437800
-93676864
-493270145
775703494
-383296819
-236624561
718915717
-547933773
917826382
584276339
-369616395
535256125
598998501
568076462
-980687456
-505583059
622015455
268801264
634210335
599839289
-270857849
-479062350
-270840886
754885574
-356766692
123339020
-883489054
34959229
48373595
-887181379
10301239
132275802
413459712
145250706
557493820
-592117271
294735874
-900290993
-755820889
105589097
174879995
104868782
522141127
-419951788
828798609
-225507423
888696777
-417669716
-171283337
-819755368
289613853
532465317
-359113850
-980551217
624798116
-471338961
-723909374
294904428
-806468166
152937961
-693368431
938031814
609872288
-470198165
822839764
685509218
-964072398
515837859
-491510613
7667985
491478534
-167063659
362342138
-743002646
134744031
162138579
-666344488
720657675
404192872
-675729409
-714254017
301518794
784823420
392890338
-572572823
-615620503
578378231
535590923
-174530273
520247985
-686513847
-444704937
-287969694
455796068
-22716155
-315695635
371140152
376863574
-114862059
-264490361
-389817333
751532523
-927087073
527585776
-533409525
191951874
391472551
719113543
436544444
198707788
-529600164
679608359
-381435565
-390076432
-710792654
-830002862
-335762089
563344455
-556127563
-146896207
611926019
69163506
786966792
190336919
211482975
-299640297
-32962305
339973812
-932258754
-954295066
192206757
684065206
793801000
756323045
420830020
364065877
656684720
970701836
844779239
836477576
-464895439
801693863
-346804781
-536910005
332313603
147341435
164455879
-156067504
-402637056
205369020
421544445
-977748484
169257846
499403064
100026898
882562273
678809998
0
