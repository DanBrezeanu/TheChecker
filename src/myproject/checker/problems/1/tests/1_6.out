-999938132
999362959
